MQTTnet.Core/Client/IMqttClientQueuedStorage.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace MQTTnet.Core.Client
+{
+    public interface IMqttClientQueuedStorage
+    {
+        Task SaveInflightMessagesAsync(IList<MqttApplicationMessage> messages);
```

Comment:
Why is this called "Inflight"? I would prefer "SaveQueuedMessagesAsync" to have it in the same way as "MqttClientQueued".

Reply:
Ok, gonna change.